cop1fab/motoly
# :heavy_plus_sign: Setting up the dev env

### What does this PR do?

This pull request adds dependences and creates a simple flask app as our boiler plate to start building! 

### How should it be tested? 

- Clone the repo by running `git clone <repo link>`,
- cd into your repo
- make sure to have virtual env installed
- create your virtual environment 
- activate it with `source virtualenv-name/bin/activate`
- run `flask run`
